From fec4f2965eaaaa4bb3535522f40cafda46446916 Mon Sep 17 00:00:00 2001
From: Zhao Jiazhong <zhaojiazhong-hf@loongson.cn>
Date: Fri, 3 Nov 2023 15:30:42 +0800
Subject: [PATCH] [mips64] Skip console-profile-wasm on unsupported platforms

Change-Id: I9ebd5529df3f9990a0434a1855a65f6b8ef45ad2
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/5003547
Commit-Queue: Zhao Jiazhong <zhaojiazhong-hf@loongson.cn>
Reviewed-by: Clemens Backes <clemensb@chromium.org>
Cr-Commit-Position: refs/heads/main@{#90743}
---
 test/inspector/inspector.status | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/test/inspector/inspector.status b/test/inspector/inspector.status
index 32e9a276450..ad52a612bd7 100644
--- a/test/inspector/inspector.status
+++ b/test/inspector/inspector.status
@@ -122,6 +122,12 @@
 
 }],  # 'arch == loong64'
 
+##############################################################################
+['arch == mips64el', {
+  # Generic wasm-to-js wrapper must be enabled.
+  'cpu-profiler/console-profile-wasm': [SKIP],
+}],  # 'arch == mips64el'
+
 ##############################################################################
 ['arch == riscv64 or arch == riscv32', {
   # SIMD support is still in progress.
-- 
2.35.1

